pages/jdl.md

```diff
@@ -1,1447 +0,0 @@
-
-You can also generate applications, entities and export them as a JDL file using [JHipster UML]({{ site.url }}/jhipster-uml/), by running `jhipster-uml your-xmi-file.xmi --to-jdl` from the root of the generated JHipster application. To learn more about JHipster UML, and install it, go to the [JHipster UML documentation]({{ site.url }}/jhipster-uml/).
-
-This can be used as a replacement to using the [entity sub-generator]({{ site.url }}/creating-an-entity/) and is the recommended approach. The idea is that it is much easier to [manage relationships]({{ site.url }}/managing-relationships/) using a visual tool than with the classical Yeoman questions and answers.
-
-The JDL project is [available on GitHub](https://github.com/jhipster/jhipster-core/), it is an Open Source project like JHipster (Apache 2.0 License). It can also be used as a node library to do JDL parsing.
-
-_If you like the [JHipster Domain Language](https://github.com/jhipster/jhipster-core/), the [JDL Studio](https://github.com/jhipster/jdl-studio/) or the [JHipster IDE](https://github.com/jhipster/jhipster-ide/) don't forget to give them a star on [GitHub](https://github.com/jhipster/) - thanks_!
-
-***
-
-Here is the JDL documentation:
-
-1. [JDL Sample](#sample)
-1. [How to use it](#howtojdl)
-   1. [Importing a JDL file](#importingjdl)
-   1. [Exporting to a JDL file](#exportingjdl)
-1. [The language](#jdllanguage)
-   1. [Application Declaration](#applicationdeclaration)
-   1. [Entity Declaration](#entitydeclaration)
-   1. [Relationship Declaration](#relationshipdeclaration)
-   1. [Enumerations](#enumerationdeclaration)
-   1. [Blobs](#blobdeclaration)
-   1. [Entity options declaration](#entityoptiondeclaration)
-   1. [Entity annotations](#annotations)
-   1. [Microservice-related options](#microserviceoptions)
-   1. [Deployment declaration](#deploymentdeclaration)
-1. [Commenting](#commentingjdl)
-1. [All the relationships](#jdlrelationships)
-1. [Constants](#constants)
-1. [Workflows](#workflows)
-1. [Annexes](#annexes)
-   1. [Available application options](#application_options)
-   1. [Available deployment options](#deployment_options)
-   1. [Available field types and constraints](#types_and_constraints)
-   1. [Available entity options](#entity_options)
-1. [Troubleshooting](#troubleshooting)
-1. [Issues and bugs](#issues)
-
-***
-
-## <a name="sample"></a> JDL Sample
-
-The Oracle "Human Resources" sample application has been translated into JDL, and is available [here](https://github.com/jhipster/jdl-samples/blob/master/Oracle-Human-Resources-sample.jdl).
-The same application is loaded by default in [JDL-Studio](https://start.jhipster.tech/jdl-studio/) and [JHipster IDE](https://www.jhipster.tech/jhipster-ide/) as well.
-
-If you're looking for more samples, there is a repository for that right [here](https://github.com/jhipster/jdl-samples).
-
-## <a name="howtojdl"></a> How to use it
-
-###  <a name="importingjdl"></a> Importing a JDL file
```

Comment:
This entire section seems to be missing, I find it quite useful may be can be included in intro or at the beginning of getting started